### ISSUE TITLE
cp: not setting the site when ?site= is on the querystring

### ISSUE THREAD

**Issue by samuelbirch** [nystudio107/craft-seomatic#1527]
On a multisite setup, when switched to a different site (not the default) in entries, then go to seomatic it will load the default site and not use the site that is passed on the querystring.

**Comment by khalwat** [comment]
Correct. Are you thinking that it should? I'm not sure if this is necessarily the behavior you'd want all the time?

**Comment by samuelbirch** [comment]
yeah thinking it should so its inline with how others are working. If you to to commerce it keeps the same site, or navigation plugin it keeps it.

Just had a client say they almost deleted stuff as they hadn't realised the site had switched.

**Comment by tibbis** [comment]
Yeah I agree, it resets the siteId all the time you click on tabs which is super-annoying when you are changing settings on each content section for a specific site (which you usually do when you add a new site or work on a site).

**Comment by khalwat** [comment]
Addressed in the above commits.

Craft CMS 3:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-seomatic": "dev-develop as 3.5.13”,
```
Then do a `composer clear-cache && composer update`

…..

Craft CMS 4:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-seomatic": "dev-develop-v4 as 4.1.10”,
```
Then do a `composer clear-cache && composer update`

…..

Craft CMS 5:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-seomatic": "dev-develop-v5 as 5.1.10”,
```
Then do a `composer clear-cache && composer update`
